test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerWithBuiltKjarIntegrationTest.java

```diff
@@ -51,7 +52,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category(Smoke.class)
+@Category({Smoke.class, JBPMOnly.class})
```

Comment:
This test class is valid for drools, please remove JBPMOnly category.
See configuration for drools tests in this class https://github.com/kiegroup/kie-cloud-tests/pull/605/files#diff-529defef45ab113c32c0e3b7339f933eR109-R118

Reply:
Thanks for spotting Jakub.
So, from the three scenarios in this test suite, two are not valid for drools. In order to avoid to initialize these two scenarios in Openshift, I will follow the same approach as other scenario builders: check the scenario in the builder constructor with:

```java
isScenarioValid();
```

This method throws an UnsupportedOperationException to skip the scenario.

Moreover, I created a new utility class ScenarioValidations.java to have methods to verify whether it's running on either drools or jbpm, and avoid that the scenario builders have to implement this verification by their own.